app/services/CategorisationService.scala

```diff
@@ -90,7 +97,7 @@ class CategorisationService @Inject() (
                                   getGoodsRecordResponse.countryOfOrigin,
                                   LocalDate.now() //TODO where does DateOfTrade come from??
                                 )
-      categorisationInfo     <- Future.fromTry(Try(CategorisationInfo.build(goodsNomenclature).get))
+      categorisationInfo     <- Future.fromTry(Try(CategorisationInfo.build(goodsNomenclature, originalCommodityCode).get))
```

Comment:
remove .get

Reply:
Done :) Committing soon :)

Reply:
Changed it, but I preferred it with the .get as it's inside a future from try anyway. Committing soon :)